internal/gapicgen/generator/git_test.go

```diff
@@ -34,3 +34,58 @@ func TestParseConventionalCommitPkg(t *testing.T) {
 		})
 	}
 }
+
+func TestFormatChanges(t *testing.T) {
+	tests := []struct {
+		name       string
+		changes    []*ChangeInfo
+		onlyGapics bool
+		want       string
+	}{
+		{
+			name:    "basic",
+			changes: []*ChangeInfo{{Title: "fix: foo", Body: "bar"}},
+			want:    "\nChanges:\n\nfix: foo\n  bar\n\n",
+		},
+		{
+			name:    "multi-lined body indented",
+			changes: []*ChangeInfo{{Title: "fix: foo", Body: "bar\nbaz"}},
```

Comment:
It might be good to add a test that includes a footer with an indented footer that has functional meaning, e.g.,:

```
feat: this is a feature
  BREAKING CHANGE: this change is breaking
```

👆 that would translate to a major bump caused by the footer commit.

Reply:
Done.